docs/README.md

```diff
@@ -2,6 +2,8 @@
 
 SaltGUI is an open source web interface for managing a SaltStack server and its minions. Built using vanilla ES6 and implemented as a wrapper around the rest_cherrypy server a.k.a. salt-api.
 
+**🔒 Security Note**: For production deployments, SSL/TLS encryption is strongly recommended. See [SSL/TLS Configuration](#ssltls-configuration) for complete setup instructions.
```

Comment:
All SSL implementations are no considered 'old'. Do we still want to mention 'SSL'? I'm always struggling with that.
Saltstack is supposed to be for serious environments, so we can assume that the audience knows about TLS and its history.
Maybe always only use "TLS" and just once (below in its paragraph) mention 'previously known as SSL' or so?